155zc.c

```diff
@@ -0,0 +1,62 @@
+typedef struct
+{
+    int capacity;
+    int topofstack;
+    int *array;
+} MinStack;
+
+/** initialize your data structure here. */
+#define EMPTYTOS (-1)
+MinStack* minStackCreate(int maxSize)
+{
+    MinStack *S;
+    S = malloc(sizeof(MinStack)*maxSize);
+    if(S == NULL)
+        return NULL;
+    S->array = malloc(sizeof(int)*maxSize);
+    if(S->array == NULL)
+        return NULL;
+    S->topofstack = EMPTYTOS;
+    S->capacity = maxSize;
+    return S;
+}
+
+void minStackPush(MinStack* obj, int x)
+{
+    if(obj->topofstack == obj->capacity)
```

Comment:
Note that in all cases you must have a space between the if and the open parenthesis. You must also have a space between the close parenthesis and the curly brace, if you're using one.